focus/operations/src/clone.rs

```diff
@@ -798,6 +801,32 @@ fn set_up_remotes(
     Ok(())
 }
 
+fn copy_dense_config(dense_repo: &Repository, sparse_repo: &Repository) -> Result<()> {
+    if cfg!(not(feature = "twttr")) {
+        return Ok(());
+    }
+
+    let dense_cfg = dense_repo.config()?.open_level(git2::ConfigLevel::Local)?;
+    let mut sparse_cfg = sparse_repo.config()?.open_level(git2::ConfigLevel::Local)?;
+
+    let keys = vec!["ci.alt.remote", "ci.alt.enabled"];
+    for k in keys {
```

Comment:
```suggestion
    for k in ["ci.alt.remote", "ci.alt.enabled"] {
```

Since this array is static, you don't even need to heap-allocate it.